docs/notifications.md

```diff
@@ -49,6 +53,8 @@ Filters use conjunctional logic.
    Trigger for notary request notifications is notary request mempool content
    change, thus, notary request event is announced every time notary request
    enters or leaves notary pool.
+ * mempool_event announcements are not bound to chain processing. Mempool events
```

Comment:
s/mempool_event/memory pool event
s/to chain/to the chain
s/Mempool events/Memory pool events